### PR TITLE
PLAT-143714: Added UI tests and extra screenshot tests for RangePicker

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [ ] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added UI tests and extra screenshot tests for RangePicker

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Note: there are no UI tests for RTL locale because the pickers behave the same as on LTR at the moment.

### Links
[//]: # (Related issues, references)
PLAT-143714

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: tests/ui/specs/RangePicker/RangePickerPage.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const {Page} = require('@enact/ui-test-utils/utils');
+const {element} = require('@enact/ui-test-utils/utils');
```
Comment (adrian-cocoara-lgp) [inline tests/ui/specs/RangePicker/RangePickerPage.js:4]:
```suggestion
const {element, Page} = require('@enact/ui-test-utils/utils');
```

**Review by adrian-cocoara-lgp — APPROVED** [review]
UI tests look good as well 👍

**Comment by daniel-stoian-lgp** [comment]
Closing in favor of #1167

**Pull request closed without merging**